### PR TITLE
updated for resubmission

### PR DESCRIPTION
**Pull request by justin98366**

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -74,15 +77,15 @@ function countriesQuestion() {
   }
   return guessCountry;
 }
-var rightCountry = countriesQuestion();
+var guessCountry = countriesQuestion();
```
Comment (Tenlia) [inline app.js:80]: you've already declared this above, you can just reassign it.

File: app.js [hunk 2]
```diff
@@ -30,41 +31,43 @@ var people = askQuestion('Do I like people? (answer yes or no)', 'You are correc
...
 
     }else{
       alert('You\'re too low!');
     }
   }
-  return rightNum;
+  return guessNum;
 }
-var rightNum = numberQuestion();
+var guessNum = numberQuestion();
```
Comment (Tenlia) [inline app.js:56]: you've already declared this once above. (You only need to declare once but you can keep reusing it)

File: app.js (hunk 1 above)
Comment (justin98366) [reply app.js:80]: The variable above is inside the function. I renamed the variable outside the function to clear up confusion

File: app.js (hunk 2 above)
Comment (justin98366) [reply app.js:56]: The variable above is inside the function. I renamed the variable outside the function to clear up confusion

**Pull request merged**